packages/chaire-lib-frontend/src/components/pages/index.ts

```diff
@@ -7,3 +7,4 @@
 export { default as LoadingPage } from './LoadingPage';
 export * from './NotFoundPage';
 export { default as LoginPage } from './LoginPage';
+export { default as MaintenancePage } from './MaintenancePage';
```

Comment:
Is this change relevant?

Reply:
Yes, it allows to import the MaintenancePage along with the rest instead of a separate import.

Reply:
ok, missed as import instead of export